docs/_articles/en/lwc/write-lwc.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Overview
+lang: en
+---
+
+The Lightning Web Components Extension for VS Code leverages the editor's built-in code navigation and language features so that you can efficiently build Lightning web components. 
+
+Take advantage of the following Lightning Web Components Extension for VS Code features:
+
+* [Code Completion](./en/lwc/write-lwc#code-completion)
+* [Linting and corrections](./en/lwc/write-lwc#linting)
+* [View Documentation on Hover](./en/lwc/write-lwc#view-documentation-on-hover)
+* [Code navigation](./en/lwc/write-lwc#code-navigation)
+
+## Code Completion
+
+The Lightning Web Components Extension builds on VS Code's language features for [HTML](https://code.visualstudio.com/docs/languages/html) and [JavaScript](https://code.visualstudio.com/docs/languages/javascript), including syntax highlighting, bracket matching, and language-specific code completions with IntelliSense. The extension provides code completion for the following Lightning Web Components resources.
+
+### `@salesforce` Scoped Modules
+
+Lightning components access Salesforce values via scoped modules. The Lightning Web Components Extension provides code completion for the following scoped modules:
+
+* `@salesforce/resourceUrl`
+* `@salesforce/contentAssetUrl`
+* `@salesforce/apex`
+* `@salesforce/user`
+
+To learn more about `@salesforce` modules, see [`@salesforce` Modules](https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.reference_salesforce_modules) in the *Lightning Web Components Developer Guide* for more information.
+
+In this example, VS Code displays the possible static resource names to choose from when completing the `import` statement.
+
+![Static Resource Completion](./images/vscode_lwc_staticresource_trailhead.png)
+
+Here's what resource completion looks like for `@salesforce/schema`. 
+
+![Schema Completion](./images/vscode_lwc_schema.png)
+
+### Lightning API 
+
+VS Code also completes Lightning API resources, such as `lightning/uiRecordApi` and `lightning/uiObjectInfoApi`. See the [`lightning/ui*Api` Wire Adapters and Functions](https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.reference_ui_api) in the *Lighting Web Components Developer Guide* for more information. 
+
+![Lightning UI Completion](./images/vscode_lwc_lightningui.png)
+
+### Lighting Web Components Syntax 
+
+VS Code completes the Lightning Web Components framework's reactive properties and wire service. See [Reactive Properties](https://developer.salesforce.com/docs/component-library/documentation/lwc/js_props_reactive) and [Use the Wire Service to Get Data](https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.data_wire_service_about) in the *Lightning Web Components Developer Guide* for more information. 
+
+Here is an example of code completion for including the Apex function `getContactList` in the `@wire` decorator. 
```

Comment:
For Apex, does the code completion get access to everything under `force-app/main/default/classes/` and would this be helpful information to include?